Framework/PythonInterface/mantid/kernel/src/Exports/UsageService.cpp

```diff
@@ -37,9 +41,23 @@ UsageServiceImpl &instance() {
   });
   return svc;
 }
+
+/// Register feature usage from a python list
+void registerFeatureUsage(UsageServiceImpl &self,
+                          const Mantid::Kernel::FeatureType &type,
+                          const object &paths, const bool internal) {
+  ExtractStdString singleString(paths);
```

Comment:
Are you referring to L49 (the use of the ExtractStdString class)? As the code stands right now I don't think it is needed - it's ended up there because I based this function on another example in the another unit. If we go ahead and add the overload to registerFeatureUsage though, I think this line will remain and will be followed by a call to singleString.check() to determine if a single string or a python list has been passed in